backend/infrahub/database/__init__.py

```diff
@@ -135,20 +136,22 @@ def __init__(
         self,
         driver: AsyncDriver,
         mode: InfrahubDatabaseMode = InfrahubDatabaseMode.DRIVER,
-        db_type: Optional[DatabaseType] = None,
-        db_manager: Optional[DatabaseManager] = None,
-        schemas: Optional[list[SchemaBranch]] = None,
-        session: Optional[AsyncSession] = None,
+        db_type: DatabaseType | None = None,
+        default_neo4j_runtime: Neo4jRuntime = Neo4jRuntime.DEFAULT,
```

Comment:
```suggestion
```

This parameter seems to be never set to the non default value, and it is a bit confusing to have both `default_neo4j_runtime` and `Neo4jRuntime.DEFAULT` so I think it should be removed